athletics.lic

```diff
@@ -71,7 +71,8 @@ class Athletics
     walk_to(@outdoorsmanship_rooms.sample) unless @outdoorsmanship_rooms.empty?
     stow_athletics_items
     wait_for_script_to_complete('outdoorsmanship', [n])
-    pause (n*15) - (Time.now - start)
+    remaining_pause = ((n*15) - (Time.now - start))
+    pause remaining_pause if remaining_pause > 0
```

Comment:
Weird that pause doesn't already do nothing for negative pauses...

Reply:
Yeah.  I am tempted to edit lich.rbw too.